kuma/users/management/commands/configure_social_auth.py

```diff
@@ -0,0 +1,92 @@
+import fileinput
+import os
+import sys
+
+from allauth.socialaccount.models import SocialApp
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+
+LOCALHOST = 'localhost:8000'
+DOMAIN = 'localhost.org'
+
+OVERWRITE_PROMPT = 'There\'s already a SocialApp for %s, if you want to overwrite it type "yes":'
+GITHUB_INFO = (
+    'Visit https://github.com/settings/developers and click "New OAuth App"\n'
+    'Set "Homepage URL" to "http://mdn.localhost:8000/" and Authorization callback URL to ' +
```

Comment:
Should this be `"http://localhost.org:8000/"` instead of `"http://mdn.localhost:8000/"`? I think we're going to settle on `localhost.org` for everything instead of `mdn.localhost`?